src/ts/ConvivaAnalytics.ts

```diff
@@ -324,6 +349,20 @@ export class ConvivaAnalytics {
 
     // Create a Conviva monitoring session.
     this.sessionKey = this.client.createSession(this.contentMetadataBuilder.build()); // this will make the initial request
+
+    // Init ad tracking with current session key
+    switch (this.config.adTrackingMode) {
+      case AdTrackingMode.AdExperience:
+        this.adTrackingModule = new AdExperienceTrackingPlugin(this.player, this.client, this.sessionKey, this.logger);
```

Comment:
are these `plugins` or `modules`? We should name the variable accordingly.

Reply:
I would go for `this.adTrackingPlugin`